src/main/java/entities/internalUserList.java

```diff
@@ -0,0 +1,17 @@
+package entities;
+
+import java.util.ArrayList;
+
+public class internalUserList {
```

Comment:
I believe @CC-3636 has this file as well as UserWhisperer removed (she no longer uses these entities anymore). Please confirm though

Reply:
correct, I removed both UserWhisperer and internalUserList because I no longer use them